.changeset/twelve-birds-wait.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/catalog-model': patch
+---
+
+Added a connection between www-artist and artist-lookup components
+
+## Screenshots
```

Comment:
Can we put this in something like this:

<details><summary>CLICK ME</summary>
<p>

#### yes, even hidden code blocks!

```python
print("hello world!")
```

</p>
</details>

So we don't clutter up the release notes as these are big images?

Reply:
Good idea. I just pushed a commit that adds details wrapper.